0-ad-history-mod-package/simulation/templates/units/han/support_trader.xml

```diff
@@ -8,6 +8,14 @@
     <Civ>han</Civ>
     <SpecificName>Shāngrén</SpecificName>
     <Icon>units/han/support_trader.png</Icon>
+    <History>
+        In the Han dynasty's early days, the Xiognu, a hostile people inhabiting the lands north of the Han empire, were repeatedly plundering the border regions. The Xiognu were nomads and rode horses, which made them extremely difficult to fight against. For that reason, the Han saw no choice but to pay tribute to the Xiognu in order to make peace. However, despite the tribute payments, the Xiognu kept on pillaging. And thanks to the Xiognu and other natural barriers, the Han were almost completely cut off from the Western part of the world.
+        When Emperor Wu planned to finally put an end to the Xiognu, he sought allies and figured the Yuezhi, another nomadic people to the west of the Xiognu, would be willing to help. The Yuezhi had recently been defeated and their king humiliated by the Xiognu. Emperor Wu sent an envoy, Zhang Qian, to try to form an alliance with the Yuezhi.
+        The mission, however, was very dangerous, as he had to travel through hostile lands. And, not long after his departure, Zhang Qian and his companions were captured by the Xiognu. Zhang Qian was surprisingly left alive and held as a captive. He took this opportunity to closely examine the Xiognu, their society, life, and fighting style. And when the chance arose, Zhang Qian fled to resume his mission. A mere ten years had passed.
+        He eventually traversed the Xiognu regions and arrived at the Yuezhi. Those were, against his expectations, not keen on starting a war against the Xiognu. Still, many of the rulers he met were open to building relations with the Han. Zhang Qian spent another year there and studied the cultures of the peoples he encountered. This region, today the Stans in Central Asia, was previously completely unknown to the Han. And, since two hundred years prior, Alexander the Great had conquered this region and established colonies there, Zhang Qian even came into contact with ancient Greek culture, a society thousands of miles away from the Han dynasty. He even saw highly priced Chinese products that had found their way to this (for the Han) exotic place.
```

Comment:
This paragraph needs to be split into two.
```suggestion
        Eventually, Zhang Qianeventually made it through the lands of the Xiognu and arrived at those of the Yuezhi. The Yuehi were, unfortunatley, not keen on starting a war against the Xiognu. Still, many of the rulers he met were open to building relations with the Han. Zhang Qian spent another year studying the cultures of the peoples he encountered.
        This region, today the Stans in Central Asia, was previously completely unknown to the Han. And, since two hundred years prior, Alexander the Great had conquered this region and established colonies there, Zhang Qian even came into contact with ancient Greek culture, a society thousands of miles away from the Han dynasty. He even saw highly priced Chinese products that had found their way to this (for the Han) exotic place.